compiler/il/symbol/OMRSymbol.hpp

```diff
@@ -432,6 +432,9 @@ class OMR_EXTENSIBLE Symbol
    void setNamedShadowSymbol();
    bool isNamedShadowSymbol();
 
+   void setImmutableField() { _flags2.set(ImmutableField); }
+   bool isImmutableField()  { return _flags.testAny(ImmutableField); }
```

Comment:
indeed a typo.   only one symbol currently sets this flag.  it is in tr.open side:  findOrCreateCurrentThread ...